Packages/Netherlands3D/BAGInformation/Runtime/BAG/GeoJSONPropertyLoader.cs

```diff
@@ -0,0 +1,114 @@
+/*
+*  Copyright (C) X Gemeente
+*                X Amsterdam
+*                X Economic Services Departments
+*
+*  Licensed under the EUPL, Version 1.2 or later (the "License");
+*  You may not use this work except in compliance with the License.
+*  You may obtain a copy of the License at:
+*
+*    https://github.com/Amsterdam/Netherlands3D/blob/main/LICENSE.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" basis,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+*  implied. See the License for the specific language governing
+*  permissions and limitations under the License.
+*/
+using Netherlands3D.Events;
+using Netherlands3D.Utilities;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.BAGInformation
+{
+	/// <summary>
+	/// Loads GeoJSON from an URL using unique ID's, and invoke events for 
+	/// returned key/value pairs for all properties.
+	/// </summary>
+	public class GeoJSONPropertyLoader : MonoBehaviour
+	{
+		[SerializeField]
+		private string idReplacementString = "{BagID}";
+
+		[Tooltip("Id replacement string will be replaced")]
+		[SerializeField]
+		private string geoJsonRequestURL = "https://service.pdok.nl/lv/bag/wfs/v2_0?SERVICE=WFS&VERSION=2.0.0&outputFormat=geojson&REQUEST=GetFeature&typeName=bag:pand&count=100&outputFormat=xml&srsName=EPSG:28992&filter=%3cFilter%3e%3cPropertyIsEqualTo%3e%3cPropertyName%3eidentificatie%3c/PropertyName%3e%3cLiteral%3e{BagID}%3c/Literal%3e%3c/PropertyIsEqualTo%3e%3c/Filter%3e";
+
+		[Header("Listen to")]
+		[SerializeField]
+		private StringListEvent loadPropertiesForIDs;
+
+		[Header("Invoke")]
+		[SerializeField]
+		private TriggerEvent loadedFeatureWithProperties;
+		[SerializeField]
+		private StringListEvent loadedPropertyKeyValue;
+
+		private Coroutine downloadProcess;
+
+		void Start()
+		{
+			loadPropertiesForIDs.started.AddListener(DownloadGeoJSONProperties);
+		}
+
+		private void OnValidate()
+		{
+			if (!geoJsonRequestURL.Contains(idReplacementString))
+			{
+				Debug.LogWarning("Make sure the url contains the id replacement string", this.gameObject);
+			}
+		}
+
+		private void DownloadGeoJSONProperties(List<string> bagIDs)
```

Comment:
function ontvangt een lijst met bagIDs, maar downloadt alleen de eerste. 
misschien beter om te laten luisteren naar een event met een enkele string ( en de naam hier ook op aanpassen?)

Reply:
yo martijn; dit is bewust al een multiselect ondersteuning. je kan nu theoretisch (als we iets soortgelijks toevoegen in de package als de Selector uit 3DAmsterdam) multiselects doen